huntsman/tests/test_camera.py

```diff
@@ -30,14 +30,16 @@
 
 # Ugly hack to access id inside fixture
 @pytest.fixture(scope='module', params=zip(params, ids), ids=ids)
-def camera(request, images_dir, camera_server):
+def camera(request, images_dir, camera_server, config_server):
     if request.param[0] == PyroCamera:
         ns = Pyro4.locateNS()
         cameras = ns.list(metadata_all={'POCS', 'Camera'})
         cam_name, cam_uri = cameras.popitem()
         camera = PyroCamera(port=cam_name, uri=cam_uri)
 
+    # Modify the client-side camera config
     camera.config['directories']['images'] = images_dir
```

Comment:
Shouldn't this be using the new `images_dir_local` fixture instead? This 'camera' is the Pyro proxy wrapper, the local interface for the distributed camera.

Actually, isn't this entirely unnecessary now that you're setting both the local and remote image directories in the config server fixture?

Reply:
I was going with the assumption that `image_dir` points to the image directory on the control computer and `image_dir_local` refers to the local device storage. So a `pyro.Camera` instance should have `self.config['directories']['images'] = images_dir`.

Although, for these tests, it doesn't matter either way because both directories are "local".

At the moment `pyro.Camera` instances don't get their config from the config server, but use  `huntsman.utils.load_config` with the default of `huntsman.yaml`. This should probably be changed...